interceptors/payment_authentication.go

```diff
@@ -3,12 +3,12 @@ package interceptors
 import (
 	"context"
 	"fmt"
+	"github.com/companieshouse/chs.go/authentication"
```

Comment:
Please configure your editor to run `go fmt` and `goimports` against each edited file